sgx/pelz_enclave.edl

```diff
@@ -0,0 +1,20 @@
+enclave {
+	from "sgx_tsgxssl.edl" import *;
```

Comment:
I think we're going to need to add:  

    from "sgx_pthread.edl" import *;

to support threading going forward.

Reply:
Done.